pkg/driver/mock/auto/securityevent.go

```diff
@@ -0,0 +1,33 @@
+package auto
+
+import (
+	"context"
+	"time"
+
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"github.com/vanti-dev/sc-bos/pkg/gentrait/securityevent"
+	"github.com/vanti-dev/sc-bos/pkg/task/service"
+)
+
+func SecurityEventAuto(model *securityevent.Model) *service.Service[string] {
+	slc := service.New(service.MonoApply(func(ctx context.Context, _ string) error {
+		go func() {
+			timer := time.NewTimer(30 * time.Second)
```

Comment:
You could just use a ticker here, can't think it's doing anything taxing that'd need the 30s to be strictly between tasks instead of periodic.